cert/openssl.cnf

```diff
@@ -129,15 +129,16 @@ basicConstraints 				= critical, CA:true
 keyUsage 						= critical, digitalSignature, cRLSign, keyCertSign
 
 ####################################################################
-#Not used yet
+
 [ crl_section ]
 
-URI.0 = http://crl3.digicert.com/WInnForumCA.crl
-URI.1 = http://crl4.digicert.com/WInnForumCA.crl
+#URI.0 = http://crl3.digicert.com/WInnForumCA.crl
+#URI.1 = http://crl4.digicert.com/WInnForumCA.crl
+URI.0 = http://localhost:9006/ca.crl
```

Comment:
Please add a note in the README that this needs to be changed before generating the certificates. Is there a way to put it in a more obvious place?

Reply:
A README   file has been added as part of the CRL server PR#469. The CRL server provides   a menu option that updates the this CRL URL with that of the CRL server and   re-generates the certificates so that the certificates have their CDP field   updated to point to the simple CRL server.